main.sh

```diff
@@ -107,40 +99,40 @@ function xml_dump {
 
 function all_sch { # arg($1:group)
     # is loaded by default so data must be loaded
-    data=($(sed 's/://g' $SCHEDULE))
```

Comment:
No, I meant to ask if you really intented to use `2utf8` here :
```bash
function batti { /usr/local/bin/2utf8 $@ | sed 's/ Group [^1]/\n&/'; }
```

I think you meant:
```bash
function batti { /usr/local/bin/batti $@ | sed 's/ Group [^1]/\n&/'; }
```